src/charm.py

```diff
@@ -225,7 +225,9 @@ def monitor_config(self) -> MongoDBConfiguration:
     def backup_config(self) -> MongoDBConfiguration:
         """Generates a MongoDBConfiguration object for backup."""
         self._check_or_set_user_password(BackupUser)
-        return self._get_mongodb_config_for_user(BackupUser, BackupUser.get_hosts())
+        return self._get_mongodb_config_for_user(
+            BackupUser, BackupUser.get_hosts(), standalone=True
```

Comment:
Should `standalone` be always set to `True` ?

Reply:
For Percona MongoDB yes, perhaps I can add a comment here to make that more clear.

from documentation:
> The pbm-agent processes should connect to their localhost mongod with a standalone type of connection.